js/markupBlog.js

```diff
@@ -0,0 +1,328 @@
+/** @format */
+const markupTitle = (value, borderClass) => {
```

Comment:
this function has a duplicate in the `[markupBlog.js](https://github.com/DenysPetrenk0/HomeWork-EPAM/pull/9/files?file-filters%5B%5D=.html&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.scss#diff-47dd2dd2367abe4a28b9fdeb220431ed34af0c925406466ade94b222bcadee30R2)` file, make it generic and move it to a separate file and import it here for using.